CHANGELOG.rst

```diff
@@ -23,6 +23,12 @@ Fixed
 Security
 ========
 
+[2.2.0] - 2018-12-14
```

Comment:
Can you please, change the kytos.json file too? In order to upgrade the version ?